Library/Formula/mcrypt.rb

```diff
@@ -1,49 +1,45 @@
-require "formula"
-
 class Mcrypt < Formula
   desc "Replacement for the old crypt package and crypt(1) command"
   homepage "http://mcrypt.sourceforge.net"
   url "https://downloads.sourceforge.net/project/mcrypt/MCrypt/2.6.8/mcrypt-2.6.8.tar.gz"
-  sha1 "8ae0e866714fbbb96a0a6fa9f099089dc93f1d86"
+  sha256 "5145aa844e54cca89ddab6fb7dd9e5952811d8d787c4f4bf27eb261e6c182098"
 
   bottle do
-    sha1 "9afdc1c3fdbf8f9801301fb959c0192b99072fc7" => :mavericks
-    sha1 "71c4c177e18d8ee3be7a1f5556c40ec5d2ba5a4d" => :mountain_lion
-    sha1 "ac147e3524ce21a9d57c0fa84e3c8e2db26321e1" => :lion
+    revision 1
   end
 
   depends_on "mhash"
 
   resource "libmcrypt" do
     url "https://downloads.sourceforge.net/project/mcrypt/Libmcrypt/2.5.8/libmcrypt-2.5.8.tar.gz"
-    sha1 "9a426532e9087dd7737aabccff8b91abf9151a7a"
+    sha256 "e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e"
   end
 
-  option :universal
-
   # Patch to correct inclusion of malloc function on OSX.
   # Upstream: https://sourceforge.net/p/mcrypt/patches/14/
   patch :DATA
 
   def install
-    ENV.universal_binary if build.universal?
-
     resource("libmcrypt").stage do
       system "./configure", "--prefix=#{prefix}",
                             "--mandir=#{man}"
-      system "make install"
+      system "make", "install"
     end
 
     system "./configure", "--prefix=#{prefix}",
                           "--with-libmcrypt-prefix=#{prefix}",
                           "--mandir=#{man}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "mcrypt", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!